routing/probability_bimodal.go

```diff
@@ -423,27 +423,31 @@ func cannotSend(failAmount, capacity lnwire.MilliSatoshi, now,
 // unbalanced channels, a very high scale assumes a random distribution. More
 // details can be found in
 // https://github.com/lightningnetwork/lnd/issues/5988#issuecomment-1131234858.
+// Additionally, we add a constant term 1/c to the distribution to avoid
```

Comment:
should we also update `P(x) ~ exp(-x/s) + exp((x-c)/s)` above to have this new constant?